packages/components/src/components/chat/UserMessage.vue

```diff
@@ -383,6 +388,13 @@ export default {
         await this.writeToClipboard(text);
       }
     },
+    saveMessage() {
+      this.$root.$emit('save-message', {
+        messageId: this.id,
+        threadId: this.threadId,
+        content: this.message,
```

Comment:
Curious, is this going to include the gatherer output? @dividedmind 

We should probably move the gatherer feedback into its own UI element and propagate that back to the frontend via the RPC. If properly presented, it would be interesting / useful to the user to see what the gatherer is doing.